xwiki-platform-core/xwiki-platform-skin/xwiki-platform-skin-skinx/src/main/java/com/xpn/xwiki/plugin/skinx/AbstractSkinExtensionPlugin.java

```diff
@@ -164,6 +180,32 @@ public Api getPluginApi(XWikiPluginInterface plugin, XWikiContext context)
         return new SkinExtensionPluginApi((AbstractSkinExtensionPlugin) plugin, context);
     }
 
+    /**
+     * @param filename The name of the file to get the URL for.
+     * @param context The current request context.
+     * @return The (unescaped) URL of the skin file.
+     * @since 14.0RC1
```

Comment:
Probably not the right version ;)

Reply:
Yep, fixed. At least I assume it will be 14.1RC1 as the fix concerns quite a lot of code and nothing is actually broken so I'm not sure this should be cherry-picked to 13.10 or 14.0.